main.go

```diff
@@ -19,13 +19,13 @@ func main() {
 
 	configFlagSet := pflag.NewFlagSet("oauth2-proxy", pflag.ContinueOnError)
 
-	// Because we parse early to determine alpha vs legacy config, we have to
+	// Because we parse early to determine config vs legacy config, we have to
 	// ignore any unknown flags for now
 	configFlagSet.ParseErrorsWhitelist.UnknownFlags = true
 
 	config := configFlagSet.String("config", "", "path to config file")
-	alphaConfig := configFlagSet.String("alpha-config", "", "path to alpha config file (use at your own risk - the structure in this config file may change between minor releases)")
-	convertConfig := configFlagSet.Bool("convert-config-to-alpha", false, "if true, the proxy will load configuration as normal and convert existing configuration to the alpha config structure, and print it to stdout")
+	legacyConfig := configFlagSet.String("legacy-config", "", "path to legacy config file")
+	convertConfig := configFlagSet.Bool("convert-legacy-config", true, "If true, the legacy toml configuration will be converted to the new yaml config structure, and print it to stdout")
 	showVersion := configFlagSet.Bool("version", false, "print version string")
```

Comment:
I might need to revert this if we don't want to break too much for a v7.6.0 release.